packages/mwp-app-server/src/util/getPlugins.js

```diff
@@ -132,6 +148,16 @@ export default function getPlugins({ languageRenderers }) {
 			headerName: CSRF_HEADER_NAME,
 			cookieName: CSRF_COOKIE_NAME,
 		}),
+		ScooterPlugin, // csp plugin (blankie) dependency
+		getCspPlugin({
+			defaultSrc: "'self' *.meetup.com beta2.dev.meetup.com:8001",
+			connectSrc: '*',
+			frameSrc: 'https://staticxx.facebook.com',
+			imgSrc: '*',
+			styleSrc: "* 'unsafe-inline'",
+			scriptSrc: "* 'unsafe-inline' 'unsafe-eval'",
+			generateNonces: 'false',
```

Comment:
These are pretty permissive. I can probably find env variables for the urls

Reply:
permissive is probably fine for now. For the dev url, you'll probably need to get the subdomain from packageConfig:

```js
import { env, package as packageConfig } from 'mwp-config';
const { subdomain } = packageConfig;
const { asset_server } = env;

...

defaultSrc = ['self', '*.meetup.com', `${subdomain}.meetup.com:${asset_server.port}`].join(' '),
```